pages/panel.js

```diff
@@ -22,11 +32,24 @@ class PanelPage extends Page {
         url={url}
         session={session}
       >
-        Panel
+        <h2>Filters</h2>
+        <div>
+          <Spinner isLoading={indicators.loading} />
+          <PanelList>
```

Comment:
What is the purpose of this component `PanelList`? Is it only a box? Why we need it?
ajja

I think that it would have a sense if you rendered the indicators inside the PanelList Component. And PanelList will receive an indicators prop